scripts/hf_eval.py

```diff
@@ -89,7 +89,7 @@ def all_linear(mod, name):
     with torch.no_grad():
         result = evaluate(
             HFLM(
-                pretrained=model.to(device),
+                pretrained=model,
```

Comment:
is this change needed

Reply:
This changeis removed from current PR, will be done in separate PR.